distributions/otelcol-contrib/.goreleaser.yaml

```diff
@@ -204,31 +219,31 @@ docker_manifests:
     image_templates:
       - otel/opentelemetry-collector-contrib:{{ .Version }}-386
       - otel/opentelemetry-collector-contrib:{{ .Version }}-amd64
-      - otel/opentelemetry-collector-contrib:{{ .Version }}-armv7
+      - otel/opentelemetry-collector-contrib:{{ .Version }}-arm/v7
```

Comment:
this changed and should probably stay as it was. same for all the other occurrences